### PR TITLE
L10N support + Simplified Chinese

### PR DESCRIPTION
**Pull request by darobin**
Support localising string that are produced by the system, include Chinese (Simplified) translations, and start making use of them.

We have Chinese users for this.

### CODE REVIEW

**Comment by marcoscaceres** [comment]
This looks good, @darobin.

File: js/w3c/l10n.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                ,   abstract:                   "摘要"
+                ,   sotd:                       "关于本文档"
+                ,   status_at_publication:      "本章节描述了本文档的发布状态。其它更新版本可能会覆盖本文档。W3C的文档列 表和最新版本可通过<a href='http://www.w3.org/TR/'>W3C技术报告</a>索引访问。"
+                ,   toc:                        "内容大纲"
+                ,   note:                       "注"
+                ,   fig:                        "圖"
+                ,   bug_tracker:                "错误跟踪："
+                ,   file_a_bug:                 "反馈错误"
+                ,   open_bugs:                  "修正中的错误"
+                ,   open_parens:                "（"
```
Comment (marcoscaceres) [inline js/w3c/l10n.js]: Why are extra spaces needed for zh?

Comment (darobin) [inline js/w3c/l10n.js]: They're not extra spaces :) It's a full-width parenthesis. It just looks like there's an extra space. Using the locale's correct punctuation is required.

Comment (marcoscaceres) [inline js/w3c/l10n.js]: ok, I wasn't sure. Thanks for clarifying :)

**Pull request merged**